### PR TITLE
PROJQUAY-872 - update pull-request template

### PR DESCRIPTION
**Pull request by thomasmckay**
**Issue:** https://issues.redhat.com/browse/PROJQUAY-872

**Changelog:** none

**Docs:** none

**Testing:** none

**Details:** Updating pull-request template to better reflect the required information for releases.

------
(_This section may be deleted._)
**All fields are required.** If a field is not applicable (eg. no relevant CHANGELOG.md), specify "none" or "n/a".

Issue: This is the PROJQUAY jira reference. Pull-request title must start with issue name "PROJQUAY-1234 - ".

Changelog: One line description to be added to CHANGELOG.md during release builds. Typically starts with "Added:", "Fixed:", "Note:", etc.

Docs: Detailed description of changes necessary to docs.projectquay.io. Examples would be addition of config.yaml, indication of UI changes and screenshot impact, and changes in behavior of features.

Testing: Detailed description of how to test changes manually. This section combined with the _Docs_ section above must be sufficiently clear for full test cases to be performed.

Details: Other information meant for pull-request reviewers and developers.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,29 +1,23 @@
...
-#### Issue: <link to story or task>
+Issue: This is the PROJQUAY jira reference. Pull-request title must start with "PROJQUAY-XXX - ".
 
+Changelog: One line description to be added to CHANGELOG.md during release builds. Typically starts with "Added:", "Fixed:", "Note:", etc.
 
-**TESTING** ->
+Docs: Detailed description of changes necessary to docs.projectquay.io. Examples would be addition of config.yaml, indication of UI changes and screenshot impact, and changes in behavior of features.
 
-**BREAKING CHANGE** ->
+Testing: Detailed description of how to test changes manually. This section combined with the _Docs_ setoin above must be sufficiently clear for full test cases to be performed.
```
Comment (kurtismullins) [inline .github/PULL_REQUEST_TEMPLATE.md:21]: `s/setoin/section`

**Comment by thomasmckay** [comment]
> It might be a little confusing to have the template fields documented at the bottom. I don't have a good solution, but perhaps if the fields are clear enough themselves then it won't require as much explanation?

I thought about putting the notes inline but it was a bit cluttered. I do want descriptions of the fields (especially to indicate that this info is _required_). Having them at the bottom is unobtrusive yet still obvious.

**Comment by BillDett** [comment]
This looks sensible to me; small enough to not slow us down but detailed enough  to be useful.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,29 +1,23 @@
+Issue: https://issues.redhat.com/browse/PROJQUAY-???
+Pull-request title must start with "PROJQUAY-??? - "
 
-### Description of Changes
```
Comment (alecmerdler) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: Why change these from headings?

Comment (thomasmckay) [reply .github/PULL_REQUEST_TEMPLATE.md:2]: As discussed, keeping them as headers just seemed too large. Wanting to keep this simple (and simple looking). Bold was a good idea.

**Pull request merged**